src/testController.ts

```diff
@@ -70,20 +70,10 @@ export class TestController {
 
     context.subscriptions.push(
       this.testController,
-      vscode.commands.registerCommand(
```

Comment:
These commands are used by the code lens, aren't they?

Reply:
Yes, that was a mistake, now restored. I had tested it but must not have realised that the extension host environment wasn't active.